amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -642,33 +635,14 @@ describe('SearchPage', () => {
   });
 
   describe('renderPopularTables', () => {
-    let content;
-    let props;
-    let wrapper;
-    beforeAll(() => {
-      const setupResult = setup({ searchTerm: '' });
-      props = setupResult.props;
-      wrapper = setupResult.wrapper;
-      content = shallow(wrapper.instance().renderPopularTables());
-    });
-    it('renders correct label for content', () => {
-      expect(content.children().at(1).find('label').text()).toEqual(POPULAR_TABLES_LABEL);
-    });
-
-    it('renders InfoButton with correct props', () => {
-      expect(content.children().at(1).find(InfoButton).props()).toMatchObject({
-        infoText: POPULAR_TABLES_INFO_TEXT,
-      });
-    });
-
-    it('renders SearchList with correct props', () => {
-      expect(content.children().find(SearchList).props()).toMatchObject({
-        results: props.popularTables,
-        params: {
-          source: POPULAR_TABLES_SOURCE_NAME,
-          paginationStartIndex: 0,
-        },
-      });
+    it('renders bookmark list and popular tables', () => {
+      const {props, wrapper} = setup();
+      expect(wrapper.instance().renderPopularTables()).toEqual(
+        <div className="search-list-container">
+          <BookmarkList />
+          <PopularTables />
+        </div>
+      );
```

Comment:
Might be better to test that `<BookmarkList>` and `<PopularTables>` exists in the result rather than expecting this exact object to be returned. This allows some flexibility while testing the core functionality. For example if someone adds an optional parameter `<BookmarkList showHeader={false}>` this test would needlessly fail.

Reply:
Agree with Daniel here, but it's a nit on my end because I know this is soon to be deleted when we add in the `HomePage`.